core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmitterImpl.java

```diff
@@ -31,6 +31,7 @@
 import com.microsoft.applicationinsights.internal.channel.TelemetrySerializer;
```

Comment:
This file has a few cases of the stack trace. Need to convert to the logger call.